lib/ohai/plugins/linux/platform.rb

```diff
@@ -32,12 +32,24 @@ def os_release_file_is_cisco?
     return false unless File.exist?('/etc/os-release')
     os_release_info = File.read('/etc/os-release').split.inject({}) do |map, key_value_line|
       key, _separator, value = key_value_line.partition('=')
-      map[key] = value
+      # strip leading/trailing quotes
+      map[key] = value.gsub(/\A"|"\Z/, '')
       map
     end
-    if os_release_info['CISCO_RELEASE_INFO'] && File.exist?(os_release_info['CISCO_RELEASE_INFO'])
-      os_release_info
+    cisco_release_info = os_release_info['CISCO_RELEASE_INFO']
```

Comment:
If I'm reading this right, this would be a lot easier to understand if cisco_release_info was something like cisco_release_file.